src/status_im2/contexts/communities/home/view.cljs

```diff
@@ -89,30 +100,38 @@
          :content-inset-adjustment-behavior :never
          :header                            [rn/view {:style (style/header-spacing top)}]
          :render-fn                         item-render
-         :data                              selected-items}])
+         :data                              selected-items
+         :on-scroll                         #(style/set-animated-card-values
+                                              {:scroll-offset (oops/oget % "nativeEvent.contentOffset.y")
+                                               :height        animated-card-height
+                                               :translation-y animated-card-translation-y
+                                               :opacity       animated-card-opacity})}])
 
      [rn/view {:style (style/blur-container top)}
       (let [{:keys [sheets]} (rf/sub [:bottom-sheet])]
         [blur/view
          {:blur-amount   (if platform/ios? 20 10)
-          :blur-type     (if (colors/dark?) :dark (if platform/ios? :light :xlight))
+          :blur-type     (theme/theme-value (if platform/ios? :light :xlight) :dark)
           :style         style/blur
           :overlay-color (if (seq sheets)
-                           (theme/theme-value colors/white colors/neutral-95-opa-70)
-                           (when (colors/dark?)
-                             colors/neutral-95-opa-70))}])
+                           (colors/theme-colors colors/white colors/neutral-95-opa-70)
+                           (theme/theme-value nil colors/neutral-95-opa-70))}])
       [common.home/top-nav {:type :grey}]
       [common.home/title-column
        {:label               (i18n/label :t/communities)
         :handler             #(rf/dispatch [:show-bottom-sheet {:content actions.home-plus/view}])
         :accessibility-label :new-communities-button
         :customization-color customization-color}]
-      [quo/discover-card
-       {:on-press            #(rf/dispatch [:navigate-to :discover-communities])
+      [:f> animated-card
+       {:style               style/card-bottom-override
+        :on-press            #(rf/dispatch [:navigate-to :discover-communities])
         :title               (i18n/label :t/discover)
         :description         (i18n/label :t/favorite-communities)
         :banner              (resources/get-image :discover)
-        :accessibility-label :communities-home-discover-card}]
+        :accessibility-label :communities-home-discover-card
+        :animated-values     {:opacity     animated-card-opacity
+                              :height      animated-card-height
+                              :translate-y animated-card-translation-y}}]
```

Comment:
The new component that is being animated, it is a wrapper for the `discovery-card`.

I added  a wrapper animated instead of modfying the card because we are just animating the container of the card, not the card itself.